cmd/signozschemamigrator/schema_migrator/logs_migrations.go

```diff
@@ -131,4 +131,64 @@ ORDER BY name ASC`,
 		},
 		DownItems: []Operation{},
 	},
+	{
+		MigrationID: 1002,
+		UpItems: []Operation{
+			AlterTableAddColumn{
+				Database: "signoz_logs",
+				Table:    "logs_attribute_keys",
+				Column: Column{
+					Name:    "timestamp",
+					Type:    DateTimeColumnType{},
+					Default: "toDateTime(now())",
+				},
+			},
+			AlterTableAddColumn{
+				Database: "signoz_logs",
+				Table:    "logs_resource_keys",
+				Column: Column{
+					Name:    "timestamp",
+					Type:    DateTimeColumnType{},
+					Default: "toDateTime(now())",
+				},
+			},
+			AlterTableModifyTTL{
+				Database: "signoz_logs",
+				Table:    "logs_attribute_keys",
+				TTL:      "timestamp + INTERVAL 15 DAY",
+				Settings: ModifyTTLSettings{
+					MaterializeTTLAfterModify: true,
```

Comment:
Shouldn't this be false? Because it reads like we want to materialise ttl after modification, but the code is disabling materialising

Reply:
oh I was using it differently in the ModifyTTLSettings, have corrected it.